health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/downloadTemplate.js

```
@@ -0,0 +1,369 @@
+        const localizedHeaders = headers.map((header) => 
+            localizationMap ? localizationMap[header] || header : header
+        );
+        
+        console.log("all boundaries 1", allBoundaries1);
+        const allDatas = getAllDatas(allBoundaries1);
+
+        console.log("all datas", allDatas);
+
+        const newLocaisationMap = localizationMap;
+
+        let localisedArray = [];
```

Biome

[error] This let declares a variable that is only assigned once.

'localisedArray' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)